otel/otel.sh

```diff
@@ -0,0 +1,79 @@
+# This initialization action installs OpenTelemetry Collector Contrib (otelcol-contrib) in a Dataproc cluster
+# configures OTel and pulls metrics from Prometheus endpoints provided by the customer. 
+# The metrics are then exported to Google Cloud Monitoring (GCM).
+
+set -euxo pipefail
+
+function install_otel() {
+  # Install otelcol-contrib as package (https://github.com/open-telemetry/opentelemetry-collector-contrib)
+  wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.81.0/otelcol-contrib_0.81.0_linux_amd64.deb
+  dpkg -i otelcol-contrib_0.81.0_linux_amd64.deb
+}
+
+function configure_endpoints() {
+  readonly PROMETHEUS_ENDPOINTS="$(/usr/share/google/get_metadata_value attributes/prometheus-scrape-endpoints || '')"
```

Comment:
All readonly variables (hardcoded or reading metadata values) can be moved to top area before functions are defined to keep consistency with other init actions.

Reply:
Done.